rest-api-spec/src/main/resources/rest-api-spec/test/indices.put_settings/11_reset.yml

```diff
@@ -9,6 +9,9 @@ setup:
         index: test-index
 ---
 Test reset index settings:
+  - skip:
+      version: " - 6.3.99"
+      reason: "include_defaults behavior is not expected to be consistent when nodes below 6.4.0 are present"
```

Comment:
looking at the goal of this test, which is to make sure that settings are resettable, I wonder if we should rather take out the last get settings with include defaults set to true. After all, the previous request already verifies that the setting was reset. I find this slightly better than skipping the whole test.